src/components/HelpTip.js

```diff
@@ -14,46 +14,65 @@ const KNOWN_HELP_DESCRIPTIONS = {
   'minimum-approval': [
     'Minimum Approval',
     <>
-      <strong>Minimum Approval</strong> is the percentage of the total token
-      supply that is required to vote “Yes” on a proposal before it can be
-      approved. For example, if the “Minimum Approval” is set to 20%, then more
-      than 20% of the outstanding token supply must vote “Yes” on a proposal for
-      it to pass.
+      The <strong>Minimum Approval</strong> is the percentage of the total token
+      supply that is required to vote “Yes” on this type of proposal before it
+      can be approved. For example, if the “Minimum Approval” is set to 20%,
+      then more than 20% of the outstanding token supply must vote “Yes” for it
+      to pass.
     </>,
   ],
   'challenge-deposit': [
     'Challenge Deposit',
     <>
-      <strong>Challenge deposit</strong> is the amount of tokens locked every
-      time a proposal is challenged. This deposit will be submitted at the time
+      The <strong>Challenge deposit</strong> is the amount of tokens locked
+      every time a proposal is challenged. This deposit is submitted at the time
       of the challenge.
     </>,
   ],
   'proposal-deposit': [
     'Proposal Deposit',
     <>
-      <strong>Proposal deposit</strong> is the amount of tokens locked every
+      The <strong>Proposal deposit</strong> is the amount of tokens locked every
       time a proposal is created. They will be automatically locked from the
       funds available in your deposit manager.
     </>,
   ],
   'settlement-period': [
     'Minimum Approval',
     <>
-      The <strong>Settlement Period</strong> is the interval of time that starts
-      when a disputable action is challenged and lasts until it’s resolved
-      between the parties (submitter and challenger), by accepting the
-      settlement offer or by raising the dispute to Celeste.
+      The <strong>Settlement Period</strong> is the amount of time the proposal
+      author has to either accept the settlement offer or raise the dispute to
+      Celeste.
     </>,
   ],
   'settlement-offer': [
     'Settlement Offer',
     <>
-      The <strong>Settlement Offer</strong> is the amount of tokens that you
-      would accept from the proposal submitter in order to cancel the action
-      without raising it to Celeste. This amount, if settled, will be slashed
-      from the proposal submitter's collateral and sent to you so it cannot be
-      greater than the collateral locked for the action but it can be zero.
+      The <strong>Settlement Offer</strong> (denominated in your Garden's token)
+      is the amount of tokens you stand to receive from the proposal author, if
+      he or she chooses to withdraw their proposal before it is raised to
+      Celeste.
```

Comment:
Do you think this bit of removed info is confusing?

`This amount, if settled, will be slashed from the proposal submitter's collateral and sent to you so it cannot be greater than the collateral locked for the action but it can be zero.`

I think we should make it clear that the settlement offer amount is taken from the author´s collateral so that users understand why the amount cannot be greater than this collateral amount. Maybe we can rephrase it ?

Reply:
I don't think the *why* matters too much?

Reply:
on this topic, i'm wondering whether we should rethink the default settlement offer...

Reply:
After reviewing your last PR i changed my mind about this. My main concern was that users know that the amount has a maximum and we clearly state that in the form. I agree that the why is not really important in this context.

Reply:
Will have a think about how we can make this clear